### PR TITLE
feat: louder RA callouts

### LINKED ISSUE #1663: [BUG] Landing height announcements too quiet
**Mod Version**

master 

**Describe the bug**

The height announcements in the landing Phase (1000, 500, ..., retard) are too quiet and should be louder. It is hard to hear because of the engine noise.  

**To Reproduce**

1. During ILS landing under 1000 ft height.
2.
3.

**Expected behavior**

announcements volume well  over engine noise 

**Actual behavior**

Volume too quiet.

**References**

**Additional context**

Was this working before/when did the issue start occurring?

Is this a problem in the vanilla unmodded game?

Discord username (if different from GitHub):

### PR DESCRIPTION
**Pull request by hotshotp**
Fixes #1663

## Summary of Changes

I made the RA altitudes callouts louder, and I made the fuel introduction sound happen only when fuel is actually introduced. Pretty small PR but it will have a huge effect now that the RA callouts are louder. Also made a small howl sound when decreasing thrust at about 40-55% n1

## Additional context

Discord username (if different from GitHub):

## Testing instructions

Spawn on approach, watch the radio altimeter and listen to the altitude callouts. Increase thrust to about 60% and then decrease to idle and listen to new howl sound.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by hotshotp** [comment]
above commits are me accidentally putting part of the flight phases from the custom AP branch, so its just me cancelling it.

**Review by tracernz — APPROVED** [review]
LGTM

**Comment by Samueleonard** [comment]
Category: QA Tester
Name: 1Samueleonard1#0001
Date of testing: Current date 28/03/2021
Version of the sim: 1.14.6.0
PR Tested: #4181
Tier of testing: 2

Changes to observe: new RA callouts + engine sound tweaks

Testing technique: Tryout
1. loaded in C+D, set up as normal
2. started engines - heard new startup sounds
3. increased/decreased thrust - heard 'howling'
4. took off, flew circuit - listened to new RA callouts

Flight notes: much louder now, perhaps too loud? would need check with real pilots

Media: n/a

Issues: just as above but good

Overall Rating: Good

Conclusions of the testing: lgtm - perhaps would need an accuracy check but works as intended

**Comment by ExampleWasTaken** [comment]
Category: QA Trial
Name: ExampleWasTaken#0886
Date of testing: 28/03/2021
Version of the sim: 1.14.6.0
PR Tested: #4181
Tier of testing: 2
Changes to observe:
louder RA callouts & newly added sound to engine rollback 

Testing technique:
1. Loaded C&D
2. Powered up
3. Spooled engines up to about 60% and back to idle
4. Checked new howling 
5. Landed to new RA callouts

Flight notes (When done a full flight): N/A

Media: N/A

Issues (If Any): N/A
Overall Rating: Good
Conclusions of the testing: LGTM

**Comment by MapleNAV-TCASFAULT** [comment]
Category: None (QA on break)
Name: Maple #3904
Date of testing: Current date 28/03/2021
Version of the sim: Version the tester uses
PR Tested: #4181 
Tier of testing: 2
Changes to observe:
- Louder RA callouts
- New howl on spool down
- Proper fuel intro sound

Testing technique: Description of how you tested the Pull Request
Steps you've made to test the Pull Request:
1.  Conducted 3 full flights with a preview of this PR from Boris
- Howl was not too noticeable on cut-to-idle
- 100 above and minimum callouts were too quiet compared to the others
2. Spawned on the runway and restarted the engines and made sure that the fuel intro "boom" would play right before egt starts rising from SAT and exactly when fuel flow goes to 160
3. Slewed to a point and tested the RA callouts including minimum callouts and tested the volume
4. Landed and made sure that the howl was noticeable

Flight notes (When done a full flight):

Media: Insert screenshots, recordings, or anything relevant to the review here. 
Feel free to compare
https://www.youtube.com/watch?v=kAEb1HvdCrM&ab_channel=SkyliteProductions
https://user-images.githubusercontent.com/72575144/112773234-31d0c800-9003-11eb-8ac8-127b8d226ebd.mp4

Issues (If Any): 100 above and minimums are maybe 1-2 decibels quieter than the RA callouts

Overall Rating: Good
Conclusions of the testing: PR works as it should, but i do find the sound to be a bit off, I think your expertise can probably figure it out. Great job nonetheless!

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/approach.FLT [hunk 1]
```diff
@@ -235,6 +235,8 @@ A32NX_DFDR_EVENT_ON = 0
 A32NX_APU_AUTOEXITING_TEST_ON = 0
 A32NX_APU_AUTOEXITING_TEST_OK = 0
 A32NX_CHRONO_ET_SWITCH_POS=1
+A32NX_CRANK_PHASE_SKIPPED=1
+
```
Comment (beheh) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/approach.FLT:239]: There's an extra line here that should not be.

**Comment by notl3ks** [comment]
Would there be a way to turn down the new RA callouts? I’m not sure about everyone else, but I find them a little too loud, to be honest.

**Comment by hotshotp** [comment]
> Would there be a way to turn down the new RA callouts? I’m not sure about everyone else, but I find them a little too loud, to be honest.

for now you can decrease the volume in the warnings slider in the msfs menu, but yeah i see i made them a little loud and ill turn them down a bit in the next update

**Comment by lionomega** [comment]
> > Would there be a way to turn down the new RA callouts? I’m not sure about everyone else, but I find them a little too loud, to be honest.
> 
> for now you can decrease the volume in the warnings slider in the msfs menu, but yeah i see i made them a little loud and ill turn them down a bit in the next update

If you can lower the volume with a slider I don't see what the problem is.
Thanks for turning up the volume!

**Comment by notl3ks** [comment]
> 
> 
> > Would there be a way to turn down the new RA callouts? I’m not sure about everyone else, but I find them a little too loud, to be honest.
> 
> for now you can decrease the volume in the warnings slider in the msfs menu, but yeah i see i made them a little loud and ill turn them down a bit in the next update

Oh than that's perfect, I didn't know you could do that. Thanks, bud!

I was thinking though, would it be possible to implement that into the speaker dial knobs next to the PFD? I noticed there's a dial to turn up/down the speaker volume which has no functionality, could maybe put that to some good use?

**Comment by wezzer** [comment]
RA callout volume is not adjustable in real aircraft, so I would avoid using the cockpit speaker volume knob for that in the sim. However, I agree that RA callout volume should be adjustable in the flight sim, but I would rather place that setting in EFB where e.g. PTU volume can also be adjusted (not adjustable in real aircraft either ;-).

**Pull request merged**